### PR TITLE
ScrollView: onReachBottom action is not fired when scrollable content bottom is not reached in pullDown mode

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/scrollView.tests.js [hunk 1]
```diff
@@ -465,26 +465,30 @@ QUnit.module('actions', moduleConfig, () => {
...
-        const $content = $scrollView.find('.' + SCROLLVIEW_CONTENT_CLASS);
-        const $bottomPocket = $scrollView.find('.' + SCROLLVIEW_BOTTOM_POCKET_CLASS);
+    [true, false].forEach((pullDownEnabled) => {
+        QUnit.test(`onReachBottom action is not fired when scrollable content bottom is not reached, ${pullDownEnabled}`, function(assert) {
+            const onReachBottomHandler = sinon.spy();
+
+            const scrollView = $('#scrollView').dxScrollView({
+                useNative: false,
+                inertiaEnabled: false,
+                onPullDown() {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets/scrollView.tests.js]: is it necessary?

File: testing/tests/DevExpress.ui.widgets/scrollView.tests.js [hunk 2]
```diff
@@ -465,26 +465,30 @@ QUnit.module('actions', moduleConfig, () => {
...
-                assert.ok(false, 'onReachBottom action should not be fired');
-            },
-            onEnd: function() {
-                assert.ok(true, 'end action was fired');
-            }
-        });
-        const $container = $scrollView.find('.dx-scrollable-container');
-        const $content = $scrollView.find('.' + SCROLLVIEW_CONTENT_CLASS);
-        const $bottomPocket = $scrollView.find('.' + SCROLLVIEW_BOTTOM_POCKET_CLASS);
+    [true, false].forEach((pullDownEnabled) => {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets/scrollView.tests.js]: seems like this test is equal for true/false, isn't it?

Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.ui.widgets/scrollView.tests.js]: https://github.com/DevExpress/DevExtreme/pull/22045/commits/15a57e89cfb4feb2abb0dce66590f3e56ec11e2f

File: testing/tests/DevExpress.ui.widgets/scrollView.tests.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.ui.widgets/scrollView.tests.js]: https://github.com/DevExpress/DevExtreme/pull/22045/commits/15a57e89cfb4feb2abb0dce66590f3e56ec11e2f

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**